web/server/codechecker_server/database/run_db_model.py

```diff
@@ -403,6 +403,20 @@ def __init__(self, name, value, description=None, user_name=None):
         self.username = user_name
 
 
+class ReportFilter(Base):
+    __tablename__ = 'report_filters'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    name = Column(String, nullable=False)
```

Comment:
Shouldn't this be product level unique?
What if the same filter name is used by multiple products?

Reply:
I agree the filter name sould be unique in the product.
When the user tries to add a filter with an exiting name, we should not allow it.

It will be necessary in order to be able to use unque filter name later in the cli.